app/scripts/config.js

```diff
@@ -0,0 +1,16 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+], function () {
+  'use strict';
+
+  var Config = {
```

Comment:
`Config` or `config`?
We aren't using `new` to create a new instance, so not sure on non-constructor naming conventions. But hey... I just work here.